src/main/java/io/microprofile/config/spi/ConvertException.java

```diff
@@ -12,24 +12,26 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- *
- * Contributors:
- *     IBM Corporation - initial API and implementation
- *     Emily Jiang - newly created class
+ * 
  *******************************************************************************/
 
-package io.microprofile.config;
-/**
- * <p> A convert to convert from String to any Java type.</p>
- * @author Emily
- *
- * @param <T> the class type to be coverted to
- */
-public interface Converter <T> {
+package io.microprofile.config.spi;
+
+public class ConvertException extends Exception {
```

Comment:
If we convert Converter to Function, then there's no need to add this exception type.